docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,104 @@
+* Change type mapping of BigQuery `timestamp` from `timestamp(3) with time zone` to `timestamp(6) with time zone` in Trino. ({issue}`9052`)
+* Fix failure for queries where predicate on `datetime` column is pushed down to BigQuery. ({issue}`9005`)
+* Fix incorrect results when using parameterized `numeric` type with non-default precision and scale. ({issue}`9060`)
+* Fix query failure when accessing tables with unsupported data type columns. ({issue}`9086`)
+* Fix failure for queries where predicate on `float64` column involving `+infinity` or 
+  `-infinity` values is pushed down to BigQuery. ({issue}`9122`)
+
+## Cassandra connector
+
+* Change minimum number of speculative executions from 2 to 1. ({issue}`9096`)
+
+## Hive connector
+
+* Support reading Parquet timestamp stored with millisecond or microsecond precision as `INT64` with 
+  logical type annotations when Hive timestamp precision is `NANOSECONDS`. ({issue}`9139`)
+* Support reading Parquet timestamp stored as `INT64` with nanosecond precision. ({issue}`9188`)
+* Support writing Parquet timestamp stored as `INT64` with nanosecond precision when experimental Parquet writer is enabled.
+  To use this, the Hive timestamp precision should be set to `NANOSECONDS`. ({issue}`9188`)
+* Support loading of S3 mappings via HTTP(S) url. The `hive.s3.security-mapping.config-file property` can now 
+  either point to a local file or a URL. ({issue}`6210`) 
+* Allow reading from tables bucketed on a column that uses type for which bucketing is not natively 
+  supported by Trino. Writing to such tables is still not allowed. ({issue}`8986`)
+* Extend set of statistics reported by JMX to include metrics for calls made to the Glue statistics API. ({issue}`9100`) 
+* Limit maximum file size generated by write operations to 1 GB by default. The limit is not exact and is applied on a best-effort basis. 
+  The limit can be set with the `hive.target-max-file-size` configuration property or the `target_max_file_size` session property. ({issue}`7991`) 
+* Allow specifying the list of schemas for which Trino will enforce that queries use a filter on partition keys for source tables.
+  The list can be specified using the `hive.query-partition-filter-required-schemas`, or the `query_partition_filter_required_schemas` session property.
+  The list is taken into consideration only if the `hive.query-partition-filter-required` configuration property or the `query_partition_filter_required` 
+  session property is set to `true`. ({issue}`9106`)
+* Fix failure when writing Parquet files with Snappy compression on ARM64. ({issue}`9148`)
+* Fix performance regression when reading Parquet files that contain dictionaries. ({issue}`9161`)
+* Fix incorrect table locking in Hive metastore when querying transactional and not partitioned table. ({issue}`9070`) 
+* Fix `ArrayIndexOutOfBoundsException` when using the experimental Parquet writer. ({issue}`5518`)
```

Comment:
Is it experimental or "native"? I'm confused which is which.

Reply:
It's both, although "native" is not a good characterization -- "optimized" is what we've called it in other contexts. In fact the description for the config option says "Experimental: Enable optimized Parquet writer".